api/HTMLElement.json

```diff
@@ -1969,10 +1969,10 @@
               "version_added": "45"
             },
             "safari": {
-              "version_added": "10"
+              "version_added": false
```

Comment:
This (`nonce`) should be `partial_implementation` for some range of versions, as WebKit as `nonce` on `HTMLLinkElement`, `HTMLScriptElement` and `HTMLStyleElement`, with a FIXME to add it to their `HTMLOrForeignElement` mixin.

Can you add custom IDL for `nonce` on those three interfaces and test how far back it goes? Maybe it's Safari 10, just claimed here :)

Reply:
Can confirm that they're on the individual elements; reverting and cherry-picking.

Reply:
Defered to #7303.